opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledgeGraph.jsx

```diff
@@ -1033,14 +1092,33 @@ class GroupingKnowledgeGraphComponent extends Component {
       });
     }, relationshipsToRemove);
     R.forEach((n) => {
-      commitMutation({
-        mutation: groupingKnowledgeGraphMutationRelationDeleteMutation,
-        variables: {
-          id: this.props.grouping.id,
-          toId: n.id,
-          relationship_type: 'object',
-        },
-      });
+      fetchQuery(groupingKnowledgeGraphCheckObjectQuery, {
+        id: n.id,
+      })
+        .toPromise()
+        .then(async (data) => {
+          if (
+            deleteObject
+            && !data.stixObjectOrStixRelationship.is_inferred
+            && data.stixObjectOrStixRelationship.groupings.edges.length === 1
```

Comment:
I saw that the behavior was the same on all containers, but is it actually the intended behavior? Shouldn't we check that there is only 1 container for the selected entity, and not only check for 1 grouping? Right now, if a relation is both in a report and in a grouping, when I cascade delete it in a grouping, it is fully deleted even though there was an other container for it (the report)